common/build.gradle

```diff
@@ -18,7 +35,9 @@ dependencies {
     // Font width data (see above)
     dddResource "dumbdogdiner:mc-font-extractor:main:mojangles_width_data@json"
 
-    implementation "com.google.code.gson:gson:2.8.6"
+    api "com.google.code.gson:gson:2.8.6"
+    compileOnly "com.google.code.gson:gson:2.8.6"
+    implementation("de.skuzzle:semantic-version:2.1.0")
```

Comment:
What is this for, aside from the blindingly obvious?

Reply:
It wouldn't compile until I did that and idk why until I funfuck gradle

Reply:
Comment refers to line 40 btw - guessing it's just a semver package but worth consulting @Prouser123 about modifying the build file if you haven't already.